### PR TITLE
Fix Jetpack Cloud verbiage

### PR DESCRIPTION
**Pull request by mattgawarecki**
#### Changes proposed in this Pull Request

* For scheduled backups in the next hour, say "In the next hour" as opposed to "In the next 1 hour"
* On the download creation confirmation page, remove superfluous "of" from instructions.
* On the Settings page, add a link to support for "Find your server credentials."
* On the "download creation failed" page, add supplementary text
* On the "restore failed" page, add supplementary instructions for what to do next
* Tweak styling on the "restore failed" page to more closely match mockups

#### Testing instructions

* Check all the aforementioned pages to ensure verbiage has changed as described and matches mockups and design feedback.

##### Mobile

image

image

##### Desktop

image

Screen Shot 2020-05-04 at 14 36 05

Fixes 1151678672052943-as-1172722058616305.

### CODE REVIEW

File: client/my-sites/activity/activity-log-item/index.jsx [hunk 1]
```diff
@@ -388,7 +388,7 @@ class ActivityLogItem extends Component {
 						disableButton={ this.state.disableDownloadButton }
 					>
 						{ translate(
-							'{{time/}} is the selected point to create a download backup of. You will get a notification when the backup is ready to download.',
+							'{{time/}} is the selected point to create a download backup. You will get a notification when the backup is ready to download.',
```
Comment (a8ci18n) [inline client/my-sites/activity/activity-log-item/index.jsx:391]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

**Comment by mattgawarecki** [comment]
In [the download flow](https://github.com/Automattic/wp-calypso/blob/fix/jetpack-cloud-verbiage/client/landing/jetpack-cloud/sections/backups/rewind-flow/download.tsx#L172), we're not currently supplying any description of what happened when the download creation process fails.

1. Do we have access to any information here that we can show to help a user?
2. If we don't, what do we want to use as our copy text?

**Comment by mattgawarecki** [comment]
After speaking with @jeffgolenski, we're going with similar copy to the "failed restore" case to handle failed downloads.

File: client/landing/jetpack-cloud/sections/backups/rewind-flow/download.tsx [hunk 2]
```diff
@@ -62,7 +62,7 @@ const BackupDownloadFlow: FunctionComponent< Props > = ( {
 			<h3 className="rewind-flow__title">{ translate( 'Create downloadable backup' ) }</h3>
 			<p className="rewind-flow__info">
 				{ translate(
-					'{{strong}}%(backupDisplayDate)s{{/strong}} is the selected point to create a download backup of. ',
+					'{{strong}}%(backupDisplayDate)s{{/strong}} is the selected point to create a downloadable backup. ',
```
Comment (a8ci18n) [inline client/landing/jetpack-cloud/sections/backups/rewind-flow/download.tsx:65]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

**Comment by mattgawarecki** [comment]
Stand by. Somehow during a rebase, I incorporated some very unrelated changes from another commit.

**Comment by mattgawarecki** [comment]
Okay, all good now.

**Comment by eeeeevon13** [comment]
Looks good to me. From the screen shots, I can't seem to get the error states to populate

**Comment by mattgawarecki** [comment]
>I can't seem to get the error states to populate

Yeah, that sounds consistent with what I see in the code. Right now I'm not actually sure it's possible to see an error state for either download or restore without forcing it through code.

**Comment by mattgawarecki** [comment]
I added a bottom margin on `.current-section` to address what @cleacos pointed out above:

image

**Comment by cleacos** [comment]
Great! LGMT

**Comment by cleacos** [comment]
Sorry, I didn't check the rest of the sections, and the margin-top added was affecting to the main Backup section:

image

Fixed on this PR: https://github.com/Automattic/wp-calypso/pull/42145

**Pull request merged**